support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -156,25 +282,88 @@ export function ThreeTierLanding(): JSX.Element {
 				borderColor="rgba(170, 170, 180, 0.5)"
 				cssOverrides={recurringContainer}
 			>
-				<h1>Support fearless, independent journalism</h1>
-				<ThemeProvider theme={buttonThemeReaderRevenueBrand}>
-					<Button
-						iconSide="left"
-						priority="primary"
-						size="default"
-						cssOverrides={checkoutBtnStyleOverrides}
-						onClick={proceedToNextStep}
-					>
-						Support with XX
-					</Button>
-				</ThemeProvider>
+				<div css={innerContentContainer}>
+					<h1 css={heading}>
+						Support fearless, <br css={tabletLineBreak} />
+						independent journalism
+					</h1>
+					<p css={standFirst}>
+						Here comes body copy about the support option below.{' '}
+						<strong>Cancel anytime.</strong>
+					</p>
+					<PaymentFrequencyButtons
+						paymentFrequencies={paymentFrequencies.map(
+							(paymentFrequency, index) => ({
+								label: paymentFrequencyMap[paymentFrequency],
+								isPreSelected: paymentFrequencies[index] === productType,
+							}),
+						)}
+						buttonClickHandler={handlePaymentFrequencyBtnClick}
+						additionalStyles={paymentFrequencyButtonsCss}
+					/>
+					<ThreeTierCards
+						cardsContent={[
+							{
+								cardTitle: 'Support',
+								currentPrice: productType === 'MONTHLY' ? '4' : '400',
+								benefits: [
+									{
+										copy: 'Regular supporter newsletter from inside our newsroom',
+									},
+									{ copy: 'See far fewer asks for support' },
+								],
+							},
+							{
+								cardTitle: 'All access digital',
+								currentPrice: productType === 'MONTHLY' ? '10' : '1000',
+								isRecommended: true,
+								benefits: [
+									{
+										copy: 'Guardian news app with personalised recommendations and offline reading',
+									},
+									{ copy: 'Ad-free reading on all your digital devices' },
+									{
+										copy: 'Regular supporter newsletter from inside our newsroom',
+									},
+									{ copy: 'See far fewer asks for support' },
+								],
+							},
+							{
+								cardTitle: 'Digital + print',
+								previousPrice: productType === 'MONTHLY' ? '25' : '2500',
+								currentPrice: productType === 'MONTHLY' ? '16' : '1600',
+								priceSuffix:
+									productType === 'MONTHLY'
+										? '£16 for the first 12 months, then £25'
+										: '£1600 for the first 12 months, then £2500',
+								benefits: [
+									{
+										copy: 'Guardian Weekly magazine delivered to your door, offering a digestible view of world news.',
+										tooltip: 'tooltip text',
+									},
+								],
+								benefitsPrefix: (
```

Comment:
yeah it does feel yucky to have to pass through markup here ... I have had a go at codifying this as part of the second schema pr ... they could be simple strings passed through if it weren't for the strong/bold parts of text. I have gone for an approach of passing through an object with copy and an optional bold property so that we can still pass through data instead of markup, but perhaps this could form part of the discussion of the second pr.